src/Stryker.Core/Stryker.Core/Mutators/LinqMutator.cs

```diff
@@ -57,57 +57,75 @@ static LinqMutator()
         public override IEnumerable<Mutation> ApplyMutations(ExpressionSyntax expr)
         {
             var original = expr;
-            if (expr.Parent is ConditionalAccessExpressionSyntax)
+            if (expr.Parent is ConditionalAccessExpressionSyntax || expr.Parent is MemberAccessExpressionSyntax)
             {
                 yield break;
             }
-            while(expr is ConditionalAccessExpressionSyntax conditional)
+
+            foreach (var mutation in FindMutableMethodCalls(expr, original))
             {
-                expr = conditional.WhenNotNull;
+                yield return mutation;
             }
+        }
 
-            if (!(expr is InvocationExpressionSyntax invocationExpression))
+        private static IEnumerable<Mutation> FindMutableMethodCalls(ExpressionSyntax expr, ExpressionSyntax original)
+        {
+            while(expr is ConditionalAccessExpressionSyntax conditional)
```

Comment:
I avoid DecendentNodes as the deep scan is non discriminated. The current logic ensures the mutant control is as close as possible to the mutant.

Let me clarify: thanks to the ternary operator, we can put the mutation control logic 'in place' with the mutant.
Except when the ternary operator leads to incorrect code, as in for statements or when mutating MemberAccess expression. 
For these, we need to have mutant control happening at a higher context than the mutation itself: with an 'if' statement for 'for' statements, or with a ternary operator at the expression level for '?.'.
Which implies having dual scanning logic: one related to where we can put the mutant control logic and one for the actual mutant.
In order to limit the amount of duplicated code the mutant control point must be as close as possible to the actual mutant. Just imagine if Stryker used 'if's at the method level to control the mutant; this would be mean we would have *n* copies of the method where *n* is the number of mutants within a method.

This means that the mutant scanning logic need to have so termination logic and not scan extensively all child nodes of the mutant control point.
Let me give you an hypothetical example for LinqMutator and the conditional operator.
Starting from this code:
```csharp
list?.Any( x => otherList?.Any( y=> x.Equals(y))
```

Current logic leads to
```csharp
Stryker.MutantControl.IsActive(0) ? 
  list?.All( x => otherList?.Any( y=> x.Equals(y)) : 
  list?.Any( x => 
         Stryker.MutantControl.IsActive(1) ? otherList?.All( y=> x.Equals(y) 
                                                                                        : otherList?.Any( y=> x.Equals(y))
```

Your alternative would lead to:
Current logic leads to
```csharp
Stryker.MutantControl.IsActive(0) ? 
  list?.All( x => otherList?.Any( y=> x.Equals(y)) : 
  Stryker.MutantControl.IsActive(1) ?
  list?.Any( x => otherList?.All( y=> x.Equals(y)) : 
  list?.Any( x => otherList?.Any( y=> x.Equals(y))
```
This is a matter of taste, but I think it is a bit difficult to tell mutants appart in the second form; but from an objective point of view, the second is one is longer. 
Plus this example remains relatively simple, one won't be able to tell longer expressions appart.

Reply:
That being said, there is probably a cleaner alternative to the current version, but I have yet to figure it out. So I would think it should be merged as is, for future refactoring when the proper solution is identified